projects/storefrontapp-e2e-cypress/cypress/integration/regression/account-profile/update-email.e2e-spec.ts

```diff
@@ -0,0 +1,67 @@
+import { login } from '../../../helpers/auth-forms';
+import { generateMail, randomString } from '../../../helpers/user';
+
+const UPDATE_EMAIL = '/my-account/update-email';
+const password = 'Password123.';
+const newUid = generateMail(randomString(), true);
+
+describe('Update email', () => {
+  before(() => {
+    cy.window().then(win => win.sessionStorage.clear());
+    cy.visit('/');
+  });
+
+  describe('when an anonymous user tries to access the page', () => {
+    it('should be redirected to the login page', () => {
+      cy.visit(UPDATE_EMAIL);
+      cy.location('pathname').should('contain', '/login');
+    });
+  });
+
+  describe('when a user is logged in', () => {
+    before(() => {
+      cy.requireLoggedIn();
+    });
+
+    beforeEach(() => {
+      cy.visit(UPDATE_EMAIL);
+    });
+
+    it('should be able to cancel and go back to home', () => {
+      cy.get('cx-update-email-form button[type="button"]').click();
+
+      cy.get('.ElectronicsHompageSplashBannerComponent').should('exist');
+
+      cy.location('pathname').should('contain', '/');
+    });
+
+    it('should be able to update its email', () => {
+      cy.get('cx-update-email-form [formcontrolname="email"]').type(newUid);
+      cy.get('cx-update-email-form [formcontrolname="confirmEmail"]').type(
+        newUid
+      );
+      cy.get('cx-update-email-form [formcontrolname="password"]').type(
+        password
+      );
+
+      cy.get('cx-update-email-form button[type="submit"]').click();
+
+      cy.get('cx-login-form').should('exist');
```

Comment:
I would like you to add a `wait` on the occ call. So wait for the update password to return.